app/main.py

```diff
@@ -1 +1,27 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = False
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return ("{" + f"Name: {self.name}, "
+                      f"Health: {self.health}, "
+                      f"Hidden: {self.hidden}" + "}")
```

Comment:
U don't need concatenation here, you can just add curly braces directly inside a string

Reply:
fix that